externals/kyuubi-jdbc-engine/src/main/scala/org/apache/kyuubi/engine/jdbc/dialect/JdbcDialect.scala

```diff
@@ -65,6 +65,8 @@ abstract class JdbcDialect extends SupportServiceLoader with Logging {
 
   def getSchemaHelper(): SchemaHelper
 
+  def initializationSQLs(): Seq[String]
```

Comment:
why not give it a default implement `SELECT 1`? then we don't need to overwrite it in each derived classes.

Reply:
> why not give it a default implementation `SELECT 1`? then we don't need to overwrite it in each derived classes.

It's a good idea to default to SELECT 1, updated later